### PR TITLE
Implement Up Next integration

### LINKED ISSUE #49: Add support for "Up Next"
The "Up Next" Kodi addon provides netflix-like binge watching features. When an episode is finished, it can show a message to play the next episode. This works since an addon can emit a message with the information about the next episode.

More information is available here: https://forum.kodi.tv/showthread.php?tid=336747

Information on how to integrate is here: https://github.com/im85288/service.upnext/wiki/Addon-Integration

### PR DESCRIPTION
**Pull request by michaelarnauts**
This PR implements Up Next integration.

Fixes #49

### CODE REVIEW

**Comment by michaelarnauts** [comment]
@dagwieers could you check if this works fine for you?

File: resources/language/resource.language.en_gb/strings.po [hunk 1]
```diff
@@ -367,18 +367,34 @@ msgctxt "#30861"
 msgid "Subtitles"
 msgstr ""
 
-msgctxt "#30863"
+msgctxt "#30862"
```
Comment (dagwieers) [inline resources/language/resource.language.en_gb/strings.po]: The reason to skip one number is that in the future the settings will allow to add a help-text.
So we have these numbers already preserved for the future.

**Comment by dagwieers** [comment]
Works well. It is nice to see this implemented using the example source code :-)

File: resources/language/resource.language.en_gb/strings.po (hunk 1 above)
Comment (michaelarnauts) [reply resources/language/resource.language.en_gb/strings.po]: I don't remember seeing this in Kodi 19. When is this expected?

I didn't want to rename all other items, so I just used the other numbers.

Comment (dagwieers) [reply resources/language/resource.language.en_gb/strings.po]: It already works in Kodi 18, you can use the new settings format. But there were still minor issues and the Kodi devs did not recommend using it.

**Pull request merged**